zipline/assets/assets.py

```diff
@@ -259,6 +259,10 @@ def _retrieve_futures_contract(self, sid):
                 data['expiration_date'] = pd.Timestamp(
                     data['expiration_date'], tz='UTC')
 
+            if data['auto_close_date']:
+                data['auto_close_date'] = pd.Timestamp(
+                    data['auto_close_date'], tz='UTC')
+
```

Comment:
maybe doesn't belong in this PR, but we could collapse the above logic to the following:
```python
 date_strs = ['start_date', 'end_date', 'first_traded',
             'notice_date', 'expiration_date', 'auto_close_date']
for param in date_strs:
    if param in data:
        data[param] = pd.Timestamp(data[param], tz='UTC')
```